content/blog/2021-04-30-april-21-community-gems.md

```diff
@@ -0,0 +1,183 @@
+---
+title: April '21 Community Gems
+date: 2021-04-30
+description: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, removing old experiments,
+  ideas for running CML pipeline reports and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, removing old experiments,
+  ideas for running CML pipeline reports  and more.
+picture: 2021-04-30/gems-cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/april-21-community-gems/
+tags:
+  - Discord
+  - Gems
+  - DVC
+  - CML
+  - Remote Storage
+---
+
+Each month we go through our Discord messages to pull out some of the best
+questions from our community. AKA: Community Gems. 💎 This month we'd like to
+thank asraniel, Julian, mattlbeck, Ahti, **\_**, yikeqicn, lexzen, EdAb,
+FreshLettuce for inspiring this month's gems!
+
+As always, [join us in Discord](https://discord.com/invite/dvwXA2N) to get all
+your DVC and CML questions answered!
+
+## DVC
+
+### [Can directories and subdirectories be created on OwnCloud with a WebDav remote when I run dvc push if they don't already exist?](https://discord.com/channels/485586884165107732/485596304961962003/831472645508694046)
```

Comment:
dvc push -> `dvc push`

Reply:
I still don't quite understand the question to be honest ... it feels that it is about specific problem that one user faced? let's discuss this in the DVC channel how can we make the title for this problem

Reply:
I think the issue was that they were using `webdav` instead of `webdavs`. But they never clarified. I think it'd be better to drop the question or alter the question to show that mismatch with https-based WebDAV remotes.

Reply:
I decided to go with a different question.